.travis.yml

```diff
@@ -88,9 +88,9 @@ matrix:
         - pip install "Jinja2>=2.10.1,<2.11"
         - pip install "intelhex>=1.3,<=2.2.1"
       script:
-        - mbedtools checkout
-        - echo mbedtools build -t GCC_ARM -m ${TARGET_NAME} -b ${PROFILE}
-        - mbedtools build -t GCC_ARM -m ${TARGET_NAME} -b ${PROFILE}
+        - mbedtools deploy
+        - echo mbedtools compile -t GCC_ARM -m ${TARGET_NAME} -b ${PROFILE}
+        - mbedtools compile -t GCC_ARM -m ${TARGET_NAME} -b ${PROFILE}
```

Comment:
The original script was using the new build tools commands as in https://github.com/ARMmbed/mbed-os-example-for-azure/blob/development/.travis.yml/#L82-L84
It should not have caused an issue in CI

Reply:
Well, the tools team has reverted the commands to align with CLI v1 (i.e. deploy, compile, etc.), so the change is correct here. Pretty much all examples including the Azure one you linked need to be fixed. @0xc0170 is aware of this and will arrange it to be done.

Reply:
Unfortunate but OK then